data/weapons/weapon_data.csv

```diff
@@ -1,59 +1,59 @@
+Magicbox PD Missile,diableavionics_magicbox,1,0.75,400,1200,,150,,3,15,5,12,0.5,3,FRAGMENTATION,25,,0.5,1,1,0,7,14,14,12,,475,35,3.7,40,,,"DO_NOT_AIM, DO_NOT_CONSERVE, PD_ONLY, ","pd9, missile0, diable_weapon_bp, rare_bp",,Diable Avionics,,Point Defense (Area),Excelent,Excelent,,,,,,,,11.23
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Thunderbolt MRM Rack,diableavionics_thunderboltsmall,0,,200,1800,,800,,5,6,6,8,,,HIGH_EXPLOSIVE,0,,0,5.5,2,0.5,6,9,9,9,,300,50,10,120,,,DO_NOT_AIM,"missile7, strike4, diable_weapon_bp",,Diable Avionics,,Finisher,Medium,Good,,,"%s: spreads up to %s of its damage inside the enemy ship, disabling weapons away from the edge of the hull.",Pierce Damage|half,,,TRUE,11.4
-Thunderbolt MRM Pod,diableavionics_thunderboltpod,1,,1150,2000,,800,,5,6,10,16,,,HIGH_EXPLOSIVE,0,,0.4,6,4,0.4,0,12,4,12,,300,650,10,120,,,DO_NOT_AIM,"missile13, strike9, diable_weapon_bp, rare_bp",,Diable Avionics,,Finisher,Medium,Good,,,"%s: spreads up to %s of its damage inside the enemy ship, disabling weapons away from the edge of the hull.",Pierce Damage|half,,,TRUE,
+Thunderbolt MRM Rack,diableavionics_thunderboltsmall,0,,200,1800,,800,,5,6,6,8,,,HIGH_EXPLOSIVE,0,,0,5.5,2,0.5,6,9,9,9,,400,50,10,250,,,DO_NOT_AIM,"missile7, strike4, diable_weapon_bp",,Diable Avionics,,Finisher,Medium,Good,,,"%s: spreads up to %s of its damage inside the enemy ship, disabling weapons away from the edge of the hull.",Pierce Damage|half,,,TRUE,11.4
+Thunderbolt MRM Pod,diableavionics_thunderboltpod,1,,1150,2500,,800,,5,6,14,24,,,HIGH_EXPLOSIVE,0,,0.4,6,4,0.4,0,12,4,12,,400,650,10,250,,,DO_NOT_AIM,"missile13, strike9, diable_weapon_bp, rare_bp",,Diable Avionics,,Finisher,Medium,Good,,,"%s: spreads up to %s of its damage inside the enemy ship, disabling weapons away from the edge of the hull.",Pierce Damage|half,,,TRUE,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Banish Torpedo (twin),diableavionics_banishsmall,1,,350,1250,,2000,2000,500,18,2,2,,,ENERGY,0,,0,10,1,,0,0,0,0,,10,120,6,300,,,"STRIKE, USE_VS_FRIGATES, DO_NOT_AIM","strike6, missile1, diable_weapon_bp",,Diable Avionics,,,Fast,Pre-Aimed,,,,,,,TRUE,9.7
-Banish Torpedo Launcher,diableavionics_banish,2,0.75,1550,1500,,2000,2000,500,25,11,6,,,ENERGY,500,,0,10,1,,0,0,0,0,,10,120,6,300,,,"STRIKE, USE_VS_FRIGATES, DO_NOT_CONSERVE, DO_NOT_AIM","missile13, strike9, diable_weapon_bp",,Diable Avionics,,Opener,Fast,Pre-Aimed,,,,,,,TRUE,
+Banish Torpedo (twin),diableavionics_banishsmall,1,,350,1250,,3000,3000,500,18,2,2,,,ENERGY,0,,0,10,1,,0,0,0,0,,10,120,6,300,,,"STRIKE, USE_VS_FRIGATES, DO_NOT_AIM","strike6, missile1, diable_weapon_bp",,Diable Avionics,,,Fast,Pre-Aimed,,,,,,,TRUE,9.7
+Banish Torpedo Launcher,diableavionics_banish,2,0.75,1550,1500,,3000,3000,500,25,11,6,,,ENERGY,0,,0,10,1,,0,0,0,0,,10,120,6,300,,,"STRIKE, USE_VS_FRIGATES, DO_NOT_CONSERVE, DO_NOT_AIM","missile13, strike9, diable_weapon_bp",,Diable Avionics,,Opener,Fast,Pre-Aimed,,,,,,,TRUE,
```

Comment:
Banish torpedo damage (both energy and EMP) increased to 3000 (from 2000).
Banish Torpedo Launcher (medium) flux cost removed (from 500).